storage_service/locations/models/package.py

```diff
@@ -317,6 +319,83 @@ def _update_quotas(self, space, location):
         location.used += self.size
         location.save()
 
+    def move_to_location(self, destination_location_uuid):
+        """
+        Move a package to another location (that has the same purpose as package's current location).
+        """
+        destination_location = Location.active.get(uuid=destination_location_uuid)
```

Comment:
Should this be checked in `resources.move_request`? It's user-submitted data, we should verify the location exists before trying to start a move and immediately erroring.